test/fixtures/v0-80212-homeowner-80000-joint-4.json

```diff
@@ -13,7 +13,7 @@
       "program_es": "Reembolsos de Electrificación y Electrodomésticos (HEAR)",
       "item": "Heat Pump Air Conditioner/Heater",
       "item_es": "Aire acondicionado/calentador con bomba de calor",
-      "more_info_url": "/app/ira-calculator/information/heat-pump-air-conditioner-heater",
+      "more_info_url": "https://homes.rewiringamerica.org/federal-incentives/25C-heat-pump-tax-credits",
```

Comment:
If I'm reading the www code correctly, this will still work there, but I can imagine an API v0 client assuming that these URLs will always be relative and always prepending `https://www.rewiringamerica.org` to absolutize it.

As far as I know, we haven't formally documented anywhere that these URLs are relative, but I think this is a breaking change in spirit if not in letter. @RandomEtc what do you think? We should probably send an email to API clients either way, but should we block on this?

Reply:
I'd propose that we just leave the v0 urls the same then and only update these for v1. it sounds like we can do that with your suggestion below of making these program URLs, and then returning the program URLs for v1, but continue using item urls for v0. I'm going to start working on that but let me know if that doesn't make sense

Reply:
Yeah, that seems fine. It hadn't quite registered for me that this is all happening in conjunction with shutting down the old calculator anyway, so we don't need to have the new URLs in v0.

Reply:
hmm, this approach isn't right either -- the new pages are really per incentive, not per program. a single IRA program could have many links associated with different incentives. incentive_item actually also isn't the right level of granularity, because one incentive item could have 2 links (one for a rebate, one for a tax credit). I think these will need to be per incentive

Reply:
I’m not totally following the mapping from old to new URLs but yes I would consider moving to an absolute URL a breaking change here. There definitely are API v0 customers who display those links (though admittedly not many, so yes it might be worth contacting them). 

I like the idea of adding a new URL field for v0 to avoid it being a breaking change, and then making sure this all makes sense in v1 before we release that publicly.